src/envoy/http/jwt_auth/jwt.cc

```diff
@@ -333,8 +333,12 @@ bool Verifier::VerifySignatureRSA(EVP_PKEY *key, const EVP_MD *md,
                                   size_t signed_data_len) {
   bssl::UniquePtr<EVP_MD_CTX> md_ctx(EVP_MD_CTX_create());
 
-  EVP_DigestVerifyInit(md_ctx.get(), nullptr, md, nullptr, key);
-  EVP_DigestVerifyUpdate(md_ctx.get(), signed_data, signed_data_len);
+  if (EVP_DigestVerifyInit(md_ctx.get(), nullptr, md, nullptr, key) != 1) {
+    return false;
+  }
+  if (EVP_DigestVerifyUpdate(md_ctx.get(), signed_data, signed_data_len) != 1) {
```

Comment:
would be good to add ENVOY_LOG to both of these with the error message from the TLS library.  See https://www.openssl.org/docs/man1.0.2/man3/ERR_get_error.html and https://www.openssl.org/docs/man1.0.2/man3/ERR_error_string.html

Reply:
I agree we should log these but I feel it's better to not to change it here as this PR is a chery-pick of https://github.com/istio/proxy/pull/2298. I think it's also better to change it in the https://github.com/google/jwt_verify_lib directly.

Reply:
Ah, I forgot I was reviewing a cherry pick.  OK, since it was approved on master and I'm not the 1.2 release manager, I'm not sure why my approval is required here.

I think it'd be good to add logging to get the error message out of the TLS library but that shouldn't block a cherry pick